share/SSTMacroConfig.cmake.in

```diff
@@ -0,0 +1,14 @@
+macro(add_sstmac_library NAME)
+  set(SST_ROOT "@sst_prefix@")
+  #ARGN is sources
```

Comment:
This comment should probably be next to `add_library` and not next to this if.  I saw `${ARGN}` and was like what is that, until I realized the comment was up here.

Reply:
Fixed.